README.md

```diff
@@ -1,3 +1,9 @@
-# aria-at-ci
+# aria-at-harness
 
 An automation tool for running tests in w3c/aria-at in continuous integration servers.
+
+## Tools
+
+- `aria-at-harness-agent` - run tests individually read from an input stream
+- `aria-at-harness-host` - run a test plan, a collection of reference files and test files, read from a client
+- `aria-at-harness-client` - read a test plan from disk, run it in a host, report result to aria-at-app
```

Comment:
Are you going to add the second two tools in a separate patch? If so, a note like "(coming soon)" will help folks understand